bin/oscapd-cli

```diff
@@ -397,11 +404,76 @@ def cli_result(dbus_iface, args):
             if args.force or confirm("Do you really want to remove result %d from task %d"
                                       % (args.result_id, args.task_id)):
                 dbus_iface.RemoveTaskResult(args.task_id, args.result_id)
+
         else:
             raise RuntimeError(
                 "Unknown result action '%s'." % (args.result_action)
             )
 
+def cli_scan(dbus_iface, args):
+    if args.fetch_cves is None:
+        fetch_cve = 2 # use defaults
+    elif args.fetch_cves:
+        fetch_cve = 1 # disable
+    else:
+        fetch_cve = 0 # enable
+
+    threads_count = 4
+
+    scan_targets = []
+
+    any_target_specified = False
+    if args.all or args.images:
+        images = json.loads(dbus_iface.images())
+        images_ids = [image["Id"].encode('ascii') for image in images]
+        scan_targets.extend(images_ids)
+        any_target_specified = True
+
+    if args.all or args.containers:
+        containers = json.loads(dbus_iface.containers())
+        container_ids = [container["Id"].encode('ascii') for container in containers]
+        scan_targets.extend(container_ids)
+        any_target_specified = True
+
+    if args.scan_targets:
+        scan_targets.extend(args.scan_targets) # todo do check if targets are valid
+        any_target_specified = True
+
+
+    if not any_target_specified:
+         raise RuntimeError(
+            "No scan target"
+        )
+
+
+    scan_results = dbus_iface.scan_list(
+        scan_targets, threads_count, fetch_cve, timeout=99999
+    )
+
+    if args.json:
+        print(scan_results)
+    else:
+        json_parsed = json.loads(scan_results)
+        if args.detail:
+
+            clean = Atomic.util.print_detail_scan_summary(
```

Comment:
I'd check `atomic_support` here even though we can't reach it with the *current* implementation. The implementation may change.